storyblok/custom-app-examples
# fix: upgrade version to manage authentication in US

Issue: EXT-1541

## What

Upgrading the Nextjs starter to be compatible with US spaces:

1. Upgrading the version of @storyblok/app-extension-auth
2. Using `storyblok-js-client` to fetch data, because it has an option to provide the region.
3. Increased type safety by validating responses.

## How to test

Try this app on a EU space and a US space:

Install with

https://app.storyblok.com/#!/install/storyblok-gmbh@app-extension-nextjs-starter

Open the Vercel preview deployment for this PR by appending `?dev=1` to the URL.

You should see something like

<img width="1512" alt="image" src="https://github.com/storyblok/custom-app-examples/assets/14206504/d4b2e843-3fc8-4e14-a84b-ca7d41b6f823">
